copy_proto_test.go

```diff
@@ -277,6 +277,72 @@ func TestStructToStruct_NonProtoFail(t *testing.T) {
 	assert.NotNil(t, err)
 }
 
+func TestStructToStruct_UnknownAnyInSrcNoSubfieldMask(t *testing.T) {
+	userWithUnknown := &testproto.User{
+		Details: []*anypb.Any{
+			{
+				TypeUrl: "example.com/example/UnknownType",
+				Value:   []byte("unknown"),
+			},
+		},
+	}
+	emptyUser := &testproto.User{}
+
+	mask := fieldmask_utils.MaskFromString("Details")
+	err := fieldmask_utils.StructToStruct(mask, userWithUnknown, emptyUser, fieldmask_utils.WithUnmarshalAllAny(false))
+	assert.NoError(t, err)
+	assert.Equal(t, userWithUnknown.Details, emptyUser.Details)
+}
+
+func TestStructToStruct_UnknownAnyInDstNoSubfieldMask(t *testing.T) {
+	userWithUnknown := &testproto.User{
+		Details: []*anypb.Any{
+			{
+				TypeUrl: "example.com/example/UnknownType",
+				Value:   []byte("unknown"),
+			},
+		},
+	}
+	emptyUser := &testproto.User{}
+
+	mask := fieldmask_utils.MaskFromString("Details")
+	err := fieldmask_utils.StructToStruct(mask, emptyUser, userWithUnknown, fieldmask_utils.WithUnmarshalAllAny(false))
+	assert.NoError(t, err)
+	assert.Equal(t, userWithUnknown.Details, emptyUser.Details)
+}
+
+func TestStructToStruct_UnknownAnyDefault(t *testing.T) {
+	userWithUnknown := &testproto.User{
+		Details: []*anypb.Any{
+			{
+				TypeUrl: "example.com/example/UnknownType",
+				Value:   []byte("unknown"),
+			},
+		},
+	}
+	emptyUser := &testproto.User{}
+
+	mask := fieldmask_utils.MaskFromString("Details")
+	err := fieldmask_utils.StructToStruct(mask, userWithUnknown, emptyUser)
+	assert.Equal(t, "proto:\u00a0not found", err.Error())
+
+}
+
+func TestStructToStruct_UnknownAnySubfieldMask(t *testing.T) {
+	userWithUnknown := &testproto.User{
+		Details: []*anypb.Any{
+			{
+				TypeUrl: "example.com/example/UnknownType",
+				Value:   []byte("unknown"),
+			},
+		},
+	}
+	emptyUser := &testproto.User{}
+
+	mask := fieldmask_utils.MaskFromString("Details{Id}")
+	err := fieldmask_utils.StructToStruct(mask, userWithUnknown, emptyUser, fieldmask_utils.WithUnmarshalAllAny(false))
+	assert.Equal(t, "proto:\u00a0not found", err.Error())
```

Comment:
Looks like this is failing for go 1.15, could you please take a look?
https://github.com/mennanov/fieldmask-utils/actions/runs/7531671531/job/20602389414?pr=40